### ISSUE TITLE
upgrade limit of scraping group

### ISSUE THREAD

**Issue by baramhs** [GangemiLorenzo/Telegram-scraper#5]
may you upgrade the scraper? so it can scrape group's that have more than 5000 members

**Comment by GangemiLorenzo** [comment]
The 5000 members limit/problem by Telethon, as stated in the readme: 
```
2. Due to a bug within Telethon scraping groups bigger than 5000 members can
lead to internal errors.
```

Anyway, I'll take a look at it and look for some workaround

**Comment by Meru3m** [comment]
@GangemiLorenzo maybe it could be worthed switching to a different library, for example 
https://docs.python-telegram-bot.org/en/v20.0a0/

I'm taking it into consideration

**Comment by GangemiLorenzo** [comment]
Yeah this library looks way more updated and valid then Telethon, I opened a branch [here](https://github.com/GangemiLorenzo/Telegram-scraper/tree/python_telegram_bot-implementation).

Please in case of PR related to this discussion, refer to this new branch.
If python-telegram-bot will result in a better solution, then we can consider to replace the main branch with this one

**Comment by baramhs** [comment]
do you have telegram acc sir ? , i want to ask something

**Comment by GangemiLorenzo** [comment]
Yep, feel free to contact me on telegram: @LGanz
